quiche/src/lib.rs

```diff
@@ -7436,6 +7430,12 @@ impl Connection {
 
         Ok(pid)
     }
+
+    // Closes the connection and drops the QLogStreamer.
+    fn close_connection_and_drop_qlog_streamer(&mut self) {
+        self.qlog.streamer = None;
```

Comment:
```suggestion
        #[cfg(feature = "qlog")]
        self.qlog.streamer = None;
```

This object is optional based on a feature, so we need to check for that